contrib/versionfile/src/mill/contrib/versionfile/VersionFileModule.scala

```diff
@@ -0,0 +1,44 @@
+package mill.contrib.versionfile
+
+import mill._, scalalib._
+
+trait VersionFileModule extends Module {
+
+  implicit val wd = os.pwd
+
+  def versionFileName = "version"
+  def versionFileDirectory = wd
+  def versionFilePath = versionFileDirectory / versionFileName
+
+  def currentVersion = T.input { Version.of(os.read(versionFilePath)) }
+  def releaseVersion = T { currentVersion().asRelease }
+  def nextVersion(bump: String) = T.command { releaseVersion().bump(bump) }
```

Comment:
```suggestion
  /** The release version. */
  def releaseVersion = T { currentVersion().asRelease }
  /** The next snapshot version. */
  def nextVersion(bump: String) = T.command { releaseVersion().bump(bump) }
```
Might as well add comments to these two too. :)